charts/tidb-cluster/values.yaml

```diff
@@ -191,10 +192,17 @@ fullbackup:
   # https://kubernetes.io/docs/tasks/job/automated-tasks-with-cron-jobs/#starting-deadline
   startingDeadlineSeconds: 3600
   # https://github.com/maxbube/mydumper/blob/master/docs/mydumper_usage.rst#options
-  options: "--rows=10000"
+  options: "--chunk-filesize=100"
   # user must set tidb cluster user and password
   user: root
   password:
+  # backup to gcp
+  gcp: {}
+    # base64 format of the service account json file data
+    # https://cloud.google.com/docs/authentication/production#obtaining_and_providing_service_account_credentials_manually
+    # and then: cat credentials.json | base64
```

Comment:
On Linux `base64` without any arguments will result in a multiline encoding string which is not allowed in `Secret`.
As stated by Kubernetes documentation https://kubernetes.io/docs/concepts/configuration/secret/
> Encoding Note: The serialized JSON and YAML values of secret data are encoded as base64 strings. Newlines are not valid within these strings and must be omitted. When using the base64 utility on Darwin/macOS users should avoid using the -b option to split long lines. Conversely Linux users should add the option -w 0 to base64 commands or the pipeline base64 | tr -d '\n' if -w option is not available.